replica/database.cc

```diff
@@ -1071,6 +1071,16 @@ std::vector<sstring>  database::get_non_system_keyspaces() const {
     return res;
 }
 
+std::vector<sstring> database::get_user_keyspaces() const {
+    std::vector<sstring> res;
+    for (auto const &i : _keyspaces) {
```

Comment:
code style nitpick: in our codebase, `&` sticks to the type, so `auto const& i` or `const auto& i`